exercises/01-getting-started.md

```diff
@@ -2,8 +2,6 @@
 In the following we describe how to get started with this project.
 You will create a new **private** repository based on this repository and run the application locally to get familiar with it.
 
-> In the session, you will be handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.
```

Comment:
I think this note might be helpful. How about rewriting it to something like this: 

> In the original session, the participants were handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.